eve/workers/pod-builder/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM centos:7
+
+ARG BUILDBOT_VERSION=0.9.12
+
+WORKDIR /home/eve/workspace
+
+RUN yum install -y epel-release
+
+RUN yum-config-manager \
+    --add-repo \
+    https://download.docker.com/linux/centos/docker-ce.repo \
+    && yum install -y \
+    gcc \
+    make \
+    python-devel \
+    python-pip \
+    genisoimage \
+    git \
+    yum-utils \
+    docker-ce-cli-18.09.2 \
+    && adduser -u 1042 --home /home/eve eve --groups docker \
+    && chown eve:eve /home/eve/workspace
+
+COPY requirements-eve.txt /tmp/
+
+RUN pip install -r /tmp/requirements-eve.txt \
+    && pip install buildbot-worker==${BUILDBOT_VERSION}
+
+USER eve
+
+CMD buildbot-worker create-worker . "$BUILDMASTER:$BUILDMASTER_PORT" "$WORKERNAME" "$WORKERPASS" \
```

Comment:
Consider switching that CMD with the following snippet in `pod.yaml`, so that the concern of starting the pod services remains with the pod file, and setting up the environment is the only concern of the Dockerfile.

```dockerfile
    args: ["-c", "buildbot-worker create-worker . ${BUILDMASTER}:${BUILDMASTER_PORT} ${WORKERNAME} ${WORKERPASS} && buildbot-worker start --nodaemon"]
```